.github/workflows/docker.yml

```diff
@@ -16,23 +17,33 @@ on:
 
 jobs:
   publish_docker_image:
-    runs-on: [self-hosted, linux-3090]
+    runs-on: ubuntu-latest
     environment: 'prod'
     env:
       TAG_PREFIX: "openmmlab/lmdeploy"
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-      - name: Get lmdeploy commit id
+      - name: Check disk space
         run: |
-          export commit_id=$(git log --format="%H" -n 1)
-          echo "commit id = $commit_id"
-          echo "COMMIT_ID=$commit_id" >> $GITHUB_ENV
+          rm -rf ${GITHUB_WORKSPACE}/.git
+          cat /proc/cpuinfo  | grep -ic proc
+          free
+          df -h
+          df . -h
+      - name: Get docker info
+        run: |
+          docker info
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
Does it expose passwords?

Reply:
No. Same use as :
https://github.com/InternLM/lmdeploy/blob/9e36648284c47fda760e11f10bb0a6968b78ae8e/.github/workflows/pypi.yml#L97

Example log info
```
Run docker/login-action@v2
  with:
    username: ***
    password: ***
    ecr: auto
    logout: true
  env:
    TAG_PREFIX: ***/lmdeploy
Logging into Docker Hub...
Login Succeeded!
```